index.js

```diff
@@ -6,7 +6,7 @@
 var client = require('redis').createClient;
 var parser = require('socket.io-parser');
 var hasBin = require('has-binary');
-var msgpack = require('msgpack-lite');
+var notepack = require('notepack.io');
```

Comment:
Since notepack is just another msgpack implementation, how about keeping the `var msgpack` here?